### ISSUE TITLE
train accuracy above 100%

### ISSUE THREAD

**Issue by rezwanh001** [huggingface/torchMoji#27]
When I am running `funetune_youtube_las.py, finetune_insults_chain-thaw.py, finetune_semeval_class-avg_f1.py` files, I am getting training accuracy above 100%. 
Why is this happening? How can I solve?

**Comment by CHULOKYEE** [comment]
I have the same problem, Do you figure out why?

**Comment by CHULOKYEE** [comment]
I have the same problem. Do you know why?
